plottr/apps/inspectr.py

```diff
@@ -477,12 +488,17 @@ def plotRun(self, runId):
         win.showTime()
 
 
-def inspectr(dbPath: str = None):
+class WindowDict(TypedDict):
+    flowchart: Flowchart
+    window: QCAutoPlotMainWindow
+
+
+def inspectr(dbPath: str = None) -> QtWidgets.QMainWindow:
```

Comment:
shouldn't we use QCodesDBInspector to be more specific? or is the idea that outside of this function the code only should care whether it is `QtWidgets.QMainWindow` or not?
```suggestion
def inspectr(dbPath: str = None) -> QCodesDBInspector:
```